### PR TITLE
chore(recording): add tooltip

### LINKED ISSUE #1481: bug(chat): Tooltip missing for Recording
**Issue:**
Like title says, tooltip is missing for the recording call button 

**screenshot:**
Screenshot 2023-11-07 at 5 03 24 PM

### PR DESCRIPTION
**Pull request by stavares843**
### What this PR does 📖

- Adds a tooltip when you start recording / stop recording

image

image

Fixes #1481

### CODE REVIEW

**Comment by phillsatellite** [comment]
Hi @stavares843 ! Looks like area for tooltip is there but no text 

Screenshot 2023-11-07 at 6 46 52 PM

File: ui/src/components/media/calling.rs [hunk 1]
```diff
@@ -70,6 +70,8 @@ pub fn CallControl(cx: Scope<Props>) -> Element {
             unmute_text: get_local_text("remote-controls.unmute"),
             listen_text: get_local_text("remote-controls.listen"),
             silence_text: get_local_text("remote-controls.silence"),
+            start_recording_text: get_local_text("remote-controls.recording"),
```
Comment (stavares843) [inline ui/src/components/media/calling.rs]:
```suggestion
            start_recording_text: get_local_text("remote-controls.start-recording"),
```

File: ui/src/components/media/calling.rs [hunk 2]
```diff
@@ -70,6 +70,8 @@ pub fn CallControl(cx: Scope<Props>) -> Element {
             unmute_text: get_local_text("remote-controls.unmute"),
             listen_text: get_local_text("remote-controls.listen"),
             silence_text: get_local_text("remote-controls.silence"),
+            start_recording_text: get_local_text("remote-controls.start-recording"),
+            stop_recording_text: get_local_text("remote-controls.recording"),
```
Comment (stavares843) [inline ui/src/components/media/calling.rs]:
```suggestion
            stop_recording_text: get_local_text("remote-controls.stop-recording"),
```

**Comment by stavares843** [comment]
THANKS @phillsatellite 😂 FIXED supposedly 😂

**Comment by phillsatellite** [comment]
@stavares843 works now! 🎉

**Comment by phillsatellite** [comment]
Proof since you dont believe me 😂 
Screenshot 2023-11-07 at 6 51 50 PM

**Comment by stavares843** [comment]
WOW 🎉 THANKS @phillsatellite 😂

**Comment by stavares843** [comment]
loool 😂

**Review by phillsatellite — APPROVED** [review]

File: ui/src/components/media/calling.rs [hunk 3]
```diff
@@ -434,20 +438,32 @@ fn ActiveCallControl(cx: Scope<ActiveCallProps>) -> Element {
...
                     rsx!(Button {
-                        icon: Icon::RadioSelected,
-                        appearance: Appearance::Secondary,
-                        onpress: move |_| {
-                            ch.send(CallDialogCmd::RecordCall);
-                        },
-                    })
+            icon: Icon::RadioSelected,
+            appearance: Appearance::Secondary,
+            tooltip: cx.render(rsx!(  // Add the tooltip for the recording button
```
Comment (stavares843) [inline ui/src/components/media/calling.rs]:
```suggestion
            tooltip: cx.render(rsx!( 
```

File: ui/src/components/media/calling.rs [hunk 4]
```diff
@@ -434,20 +438,32 @@ fn ActiveCallControl(cx: Scope<ActiveCallProps>) -> Element {
                     rsx!(Button {
                         icon: Icon::StopCircle,
                         appearance: Appearance::Danger,
-                        onpress: move |_| {
-                            ch.send(CallDialogCmd::StopRecording);
-                        },
-                    })
+                        tooltip: cx.render(rsx!(  // Add the tooltip for the recording button
```
Comment (stavares843) [inline ui/src/components/media/calling.rs]:
```suggestion
                        tooltip: cx.render(rsx!( 
```

File: ui/src/components/media/calling.rs [hunk 5]
```diff
@@ -434,20 +438,32 @@ fn ActiveCallControl(cx: Scope<ActiveCallProps>) -> Element {
...
                     rsx!(Button {
-                        icon: Icon::RadioSelected,
-                        appearance: Appearance::Secondary,
-                        onpress: move |_| {
-                            ch.send(CallDialogCmd::RecordCall);
-                        },
-                    })
+            icon: Icon::RadioSelected,
+            appearance: Appearance::Secondary,
+            tooltip: cx.render(rsx!(
```
Comment (stavares843) [inline ui/src/components/media/calling.rs]:
```suggestion
            tooltip: cx.render(rsx!(
```

File: ui/src/components/media/calling.rs [hunk 6]
```diff
@@ -434,20 +438,32 @@ fn ActiveCallControl(cx: Scope<ActiveCallProps>) -> Element {
                     rsx!(Button {
                         icon: Icon::StopCircle,
                         appearance: Appearance::Danger,
-                        onpress: move |_| {
-                            ch.send(CallDialogCmd::StopRecording);
-                        },
-                    })
+                        tooltip: cx.render(rsx!(
```
Comment (stavares843) [inline ui/src/components/media/calling.rs]:
```suggestion
                        tooltip: cx.render(rsx!(
```

**Review by dariusc93 — APPROVED** [review]

**Pull request merged**